package-dev/Runtime/SentryInitialization.cs

```diff
@@ -107,19 +107,60 @@ public string Platform
         public Il2CppMethods Il2CppMethods => _il2CppMethods;
 
         private Il2CppMethods _il2CppMethods
-// Lowest supported version to have all required methods below
+            // Lowest supported version to have all required methods below
 #if !ENABLE_IL2CPP || !UNITY_2020_3_OR_NEWER || !UNITY_64
             ;
 #else
             = new Il2CppMethods(
                 il2cpp_gchandle_get_target,
-#if UNITY_2021_3_OR_NEWER
-                il2cpp_native_stack_trace,
-#else
                 Il2CppNativeStackTraceShim,
-#endif
                 il2cpp_free);
 
+#pragma warning disable 8632
+        // The incoming `IntPtr` is a native `char*`, a pointer to a
+        // nul-terminated C string. This function converts it to a C# string,
+        // and also byte-swaps/truncates on ELF platforms.
+        private static string? SanitizeDebugId(IntPtr debugIdPtr)
+        {
+            if (debugIdPtr == IntPtr.Zero)
+            {
+                return null;
+            }
+
+#if UNITY_ANDROID || UNITY_STANDALONE_LINUX
```

Comment:
just checking - is there any diference between 32 and 64-bit android with regards to these UUIDs